webrtc.html

```diff
@@ -6645,35 +6660,67 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
+              signal the system could possibly encode, and 127 represents
+              silence.</p>
+            </dd>
+          </dl>
+        </section>
+      </div>
+      <div>
+        <pre class="idl">interface RTCRtpSynchronizationSource {
+    readonly        attribute DOMHighResTimeStamp timestamp;
+    readonly        attribute unsigned long       source;
+    readonly        attribute byte                audioLevel;
+    readonly        attribute boolean?            voiceActivityFlag;
+};</pre>
+        <section>
+          <h2>Attributes</h2>
+          <dl data-link-for="RTCRtpSynchronizationSource" data-dfn-for=
+          "RTCRtpSynchronizationSource" class="attributes">
+            <dt><dfn><code>timestamp</code></dfn> of type <span class=
+            "idlAttrType"><a>DOMHighResTimeStamp</a></span>, readonly</dt>
+            <dd>
+              <p>The timestamp of type DOMHighResTimeStamp [[!HIGHRES-TIME]],
+              indicating the most recent time of playout of an RTP packet
+              from the source. The timestamp is defined in
+              [[!HIGHRES-TIME]] and corresponds to a local clock.</p>
+            </dd>
+            <dt><dfn><code>source</code></dfn> of type <span class=
+            "idlAttrType"><a>unsigned long</a></span>, readonly</dt>
+            <dd>
+              <p>The SSRC identifier of the synchronization source.</p>
             </dd>
             <dt><dfn><code>audioLevel</code></dfn> of type <span class=
             "idlAttrType"><a>byte</a></span>, readonly , nullable</dt>
             <dd>
               <p>The audio level contained in the last RTP packet played from
-              this source. If the source corresponds to an SSRC,
-              <code>audioLevel</code> will be the level value defined in
-              [[!RFC6464]], if the RFC 6464 header extension is present. If the
-              RFC 6464 extension header is not present, the browser will
-              compute a value for <code>audioLevel</code> as if it had come
-              from RFC 6464. If the source corresponds to a CSRC,
-              <code>audioLevel</code> will be the level value defined in
-              [[!RFC6465]] if the RFC 6465 header extension is present, and
-              otherwise <code>null</code>. RFC 6464 and 6465 define the level
-              as a integral value from 0 to 127 representing the audio level in
-              negative decibels relative to the loudest signal that the system
-              could possibly encode. Thus, 0 represents the loudest signal the
-              system could possibly encode, and 127 represents silence.</p>
+              this source. <code>audioLevel</code> will be the level value
+              defined in [[!RFC6464]], if the RFC 6464 header extension is
+              present. If the RFC 6464 extension header is not present, the
+              browser will compute a value for <code>audioLevel</code> as if it
+              had come from RFC 6464.
```

Comment:
Hopefully not, since it's already computed for stats. Though I'm not changing that in this PR, just moving it around.